packages/taiko-client/driver/anchor_tx_constructor/anchor_tx_constructor.go

```diff
@@ -19,8 +20,8 @@ import (
 
 var (
 	// Each TaikoL2.anchor transaction should use this value as it's gas limit.
-	AnchorGasLimit     uint64 = 250_000
-	GoldenTouchAddress        = common.HexToAddress("0x0000777735367b36bC9B61C50022d9D0700dB4Ec")
+	AnchorGasLimit     = taiko.AnchorGasLimit
```

Comment:
Then there is no need to define `AnchorGasLimit ` and `GoldenTouchAddress ` variables anymore, lets just `consensus.AnchorGasLimit ` / `consensus.GoldenTouchAddress`